string-manipulation-2/src/num-vowels.js

```diff
@@ -0,0 +1,16 @@
+/* exported numVowels */
+/*
+make the string lowercase
+create a variable for the number of vowels counted, start at 0
+look at each character in the string and compare it to the condition of vowels
+add one to the vowels counted each time a vowel is found
+return the value of all the vowels counted */
+function numVowels(string) {
+  var vowelCount = 0;
+  var newString = string.toLowerCase();
+  for (var i = 0; i < string.length; i++) {
+    if (newString[i] === 'a' || newString[i] === 'e' || newString[i] === 'i' || newString[i] === 'o' || newString[i] === 'u') {
+      vowelCount++;
+    }
+  } return vowelCount;
```

Comment:
Make sure your `return` statement is always on its own line, it should not be on the same line as a closing brace.